src/scantree/test_utils.py

```diff
@@ -4,20 +4,37 @@
 from ._path import DirEntryReplacement, RecursionPath
 
 
+def convert_path(path: str) -> str:
```

Comment:
Cool, I guess we could have replaced the plain string paths with `os.path.join(...)` as well, but this works well 👍

Reply:
yeah, but it was not sure it will work for already complete paths stored as strings